File: 1791-find-center-of-star-graph/1791-find-center-of-star-graph.java
class Solution {
    public int findCenter(int[][] edges) {
    
  int c1 = edges[0][0];
  int c2 = edges[0][1];


    if(edges[1][0] == c1 || edges[1][1] == c1)
        return c1;
    else
        return c2;
    
    
}
}

File: 1791-find-center-of-star-graph/1791-find-center-of-star-graph.cpp
class Solution {
public:
    int findCenter(vector<vector<int>>& edges) {
        int u1=edges[0][0];
        int v1=edges[0][1];
        int u2=edges[1][0];
        int v2=edges[1][1];
        if(u1==u2)return u1;
        if(u1==v2)return u1;

        if(v1==u2)return u2;
        return v1;
    }
};